apiv2/database/migrations/versions/20250819_111511_autogenerated.py

```diff
@@ -3,36 +3,38 @@
 Create Date: 2025-08-19 15:15:14.066135
 
 """
+
 import sqlalchemy as sa
 from alembic import op
 
 # revision identifiers, used by Alembic.
-revision = '20250819_111511'
```

Comment:
Does this file get formatted by something? If possible maybe we can completely revert this file just so as not to pollute the diff, but no worries if we can't